### PR TITLE
Add keda file

### PR DESCRIPTION
**Pull request by odidev**
Added keda.md file
Signed-off-by: odidev <odidev@puresoftware.com>

### CODE REVIEW

File: content/sw_database/keda.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        official_docs: https://keda.sh/docs/2.13/deploy/
+    arm_recommended_minimum_version: 
+        version_number: 
+        release_date: 
+
+  
+optional_hidden_info:
+    release_notes__supported_minimum: https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v270
+    release_notes__recommended_minimum: 
+    other_info: Binaries are not available for both AMD64 and ARM64 platforms. Releasing Dockerimage for both ARM64 and AMD64 from version 2.7.0.
```
Comment (odidev) [inline content/sw_database/keda.md:28]: Provide link for the docker images.

File: content/sw_database/keda.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    release_date: 05/05/2022 
+
+  
+optional_info:
+    homepage_url: https://keda.sh/
+    support_caveats: 
+    alternative_options: 
+    getting_started_resources: 
+        arm_content:
+        partner_content:  https://aws.amazon.com/blogs/compute/mixing-aws-graviton-with-x86-cpus-to-optimize-cost-and-resilience-using-amazon-eks/
```
Comment (odidev) [inline content/sw_database/keda.md:18]: remove extra space.

File: content/sw_database/keda.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+--- 
+name: Keda 
+category: Installable Platform
+description:  KEDA is a Kubernetes-based Event Driven Autoscaling component. It provides event driven scale for any container running in Kubernetes
```
Comment (odidev) [inline content/sw_database/keda.md:4]: remove extra space.

File: content/sw_database/keda.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        official_docs: https://keda.sh/docs/2.13/deploy/
+    arm_recommended_minimum_version: 
+        version_number: 
+        release_date: 
+
+  
+optional_hidden_info:
+    release_notes__supported_minimum: https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v270
+    release_notes__recommended_minimum: 
+    other_info: Binaries are not available for both AMD64 and ARM64 platforms. Releasing Dockerimage for both ARM64 and AMD64 from version [2.7.0](https://github.com/kedacore/keda/pkgs/container/keda/21356119?tag=2.7.0)
```
Comment (pruthviteja) [inline content/sw_database/keda.md:28]: Please add space between "Dockerimage"

**Pull request merged**